src/utils/surfaces.py

```diff
@@ -37,5 +52,8 @@
     ElementType.CASTLE: {ElementSubType.DEFAULT_CASTLE: []},
     ElementType.FLAG: {ElementSubType.DEFAULT_FLAG: []},
     ElementType.MOUNTAIN: {ElementSubType.DEFAULT_MOUNTAIN: []},
-    ElementType.COIN: {ElementSubType.DEFAULT_COIN: []},
+    ElementType.MISTERY_BOX: {
+        ElementSubType.MASTERY_BLOCK: [MASTERY_BLOCK_1, MASTERY_BLOCK_2]
+    },
+    ElementType.COIN: {ElementSubType.COIN: [COIN_1, COIN_2, COIN_3, COIN_4]}
 }
```

Comment:
This pr only has the new assets of blocks, this images are compressed? This images has a fixed correct size? (If thats correct, document all the sizes for those assets) and add a capture of how all of these already works in the game

Reply:
If this PR is only blocks, COINS would also count as blocks, because in the game they have interactions and can be displayed on the map.

Also all images are being compressed as much as possible.

But the size of the images are staying as they were uploaded, because if the need to change to larger or smaller sizes comes, by keeping the size fixed, it will end up ruining the images to the point of making them blurry, but the sizes of each image or sprite would be defined within the CONSTANT, the sizes that are at this moment of each image will not bother in any way in the performance of the game.

Reply:
The wiki of the sizes: https://github.com/software-development-v/mario-pygame/wiki/SPRITES-&-ASSETS-SIZES